### DISCUSSION TITLE
Profile Sharing!

### DISCUSSION THREAD

**Discussion by rmroc451** [nguyenquyhy/Flight-Tracker-StreamDeck#128] [Show and tell]
Please post/upload your profiles along with images so folks know what to expect when they download them.

Please also note what version of the app you last used it with, and what planes you usually use it with.

We are excited to see your creations!

**Comment by Crunchmeister1** [comment]
This isn't a full profile, but a simple profile with 2 folders for the G1000 PFD and MFD. You can copy and past these icons out of this profile into your own.

Requires:
- Flight Tracker plugin 0.23 
- MobiFlight plugin set up and working
- Stream Deck XL (uses all buttons).


[G1000 PDF and MFD Icons.zip](https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/files/5760938/G1000.PDF.and.MFD.Icons.zip)

**Reply by brlowe1965** [reply]
Thank you

**Reply by Crunchmeister1** [reply]
Glad you find it useful. I know I do!

**Reply by Tuvaka** [reply]
how do you install these?

**Comment by johans1234** [comment]
Here my profiles for:
Carenado M20R Ovation (FS2020)
Carenado PA44 Seminole (FS2020)
Cessna 172 (FS2020)

[Profiles-OLDSURE72.zip](https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/files/5900417/Profiles-OLDSURE72.zip)

**Reply by beau76** [reply]
HI @johans1234 
I am using your c172 profile is the anyway I can get your Icon pack
how did you get the green tint behind the image
thanks, beau

**Reply by johans1234** [reply]
here you find a zip with all my Icons and a ppt with the editable versions of the icons
https://app.box.com/s/wat0qjxwzvnqpwh45j8qwdhnx3wi50eo

**Reply by Schlicki** [reply]
Der Spickzettel ist aber auch gut :)

**Reply by johans1234** [reply]
wenn ich wieder mehr zum Fliegen komme, gibt es auch wieder mehr Videos und solches Beiwerk (follow me on YouTube - Oldsure72)

**Reply by Schlicki** [reply]
Gerne :)

Bei Interesse -> quiet pilot https://www.twitch.tv/pommfriedel

**Reply by nostradimoja** [reply]
Hello and thank you can you integrate a button with the avionics (2 white) button in the cessna 172
 PS: I am a graphic designer if you need me I am available
******************************
Bonjour et merci pouvez-vous intégré un bouton avec l'avionique merci bouton dans le cessna 172
 PS:  je suis graphiste si vous avez besoin de moi je suis disponible

**Comment by OzMaverick777** [comment]
I posted a Cessna 208B profile with working Garmin with Mobiflight https://flightsim.to/file/7329/stream-deck-5x3-cesena-208b-profile-with-garmin

**Comment by VVingnut** [comment]
G'day Everyone - I'm a little unsure how to *update* the StreamDeck addon once I've downloaded the new version.  Any assistance would be greatly appreciated.

VVingnut

**Comment by Wings64** [comment]
I'm almost finished developing a Caravan folder for the XL deck inside my FS2020 profile.
I should have started it as a separate profile, but little things grew larger!
Any hints how to move nested folders more easily than one key at a time into a fresh profile with the initial view an open rather than closed folder so I can share?

**Comment by nok3** [comment]
Hi, this is my profile. which should evolve. I use my Stream Deck as an indicator and shortcut for some controls, such as flaps and spoiler or landing gear. because I have not yet hotas.

Other planes should make their appearance 

As a picture is better than a long speech 

<img src="https://cdn.flightsim.to/images/14/RfdW0FKh.jpg?width=1400&auto_optimize=medium" height="360px" width="640px" />

<img src="https://cdn.flightsim.to/images/14/mkDUgJEC.png?width=1400&auto_optimize=medium" height="360px" width="640px" />

https://fr.flightsim.to/file/13562/stream-deck-3x15-for-flight-simulator-flight-tracker-free-a320-for-the-time-being

Cordially

**Reply by Schlicki** [reply]
How do I engage a selected ALT?

**Reply by jgyo** [reply]
Your graphics are top-notch!

**Comment by beau76** [comment]
Hi I have a question about the g1000 PFD and MFD
what do the number keys  1 to 12 represent thanks

**Reply by rmroc451** [reply]
Those correspond to the soft keys below the MFD/PFD (from left to right, 1 - 12) 😀

**Reply by beau76** [reply]
@rmroc451 thanks for the info

**Comment by N6722C** [comment]
I'm looking for some  + & - Button functionality to control Panel & Cockpit Light intensity .. Say for a C172  .
Anyone got a solution to this,  that ideally does not need a lot of script writting  ..

**Comment by neoteknic** [comment]
I'm starting a stream deck profile library sharing (just online today) : 
https://stream-deck-profiles-library.com
Still need some work (software, tags, categories, email alert for update, etc...), but upload/ download/ tags are working
You are welcome to share your profiles here !
@nok I have "forked" your profile for stream deck XL

**Reply by nok** [reply]
I'm the wrong guy - ping @nok3 😄 .

**Reply by nok3** [reply]
Seriously !? you have no qualms about appropriating the work of others, without asking for permission first?

**Comment by phenDruck** [comment]
Thank you Neoteknic!!!  This will be a big help for noobs like me!!!!

"Democracy is two wolves and a lamb voting on what to have for lunch. Liberty is a well-armed lamb contesting the vote." -Ben Franklin

**Comment by Wings64** [comment]
Hi,

 

With G1000 NXI mod installed, the Preset Toggle Button set to Altitude in the Grand Caravan model, the indicated altitude shows 99000. I get the impression that the access variable has changed since it seems OK elsewhere.

 

The on/off images work still as expected and other features seem to be OK still.

 

I haven’t found a way round this. Can you help?

 

Thanks as always for your important software.

 

David J

**Comment by scuba1976** [comment]
Hello.
Some of you could set GNT750 with streamdeck?
I did this profile to use PFD e MFD with c172. I used icons e parts from yours profile, but i cannot do the same with gnt750
Thanks to all

[G1000 profile.zip](https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/files/7804019/G1000.profile.zip)

**Comment by jgyo** [comment]
I have created a Generic Aircraft profile that should be useful as-is for many GA planes. My idea when creating it was to make it so that it could be copied and modified for use with specific aircraft. This profile includes folders for AP, flaps, switches, pushback, communications, gauges, and cameras.

Here are a few screenshots from the profile.

This is the main folder:
![Snag_1568c804](https://user-images.githubusercontent.com/2886615/150619379-81d38b12-06d5-475f-9e66-4aa166b31dc8.png)

Then we have the AP controls:
![Snag_156a29d6](https://user-images.githubusercontent.com/2886615/150619419-b966feb1-e20e-40cf-818e-c14b8a621f7e.png)

Flaps:
![Snag_156ac328](https://user-images.githubusercontent.com/2886615/150619433-b1b4c9a3-714f-4d72-995c-7ad82f5c3d11.png)

Switches:
![Snag_156b5cf7](https://user-images.githubusercontent.com/2886615/150619449-f3bf7c48-9b83-4d69-9069-407950610e01.png)

The pushback folder needs a little work but does work mostly in its current state.
![Snag_156cb7b5](https://user-images.githubusercontent.com/2886615/150619503-40ffa6de-7bd0-4340-bd4a-9066f8dcb466.png)

Communications:
![Snag_156d5889](https://user-images.githubusercontent.com/2886615/150619526-5661a24d-e36b-444d-bbb2-6ddd754ee867.png)

The gauges folder needs a lot of work:
![Snag_156e1b8a](https://user-images.githubusercontent.com/2886615/150619561-59d8a726-f9bc-4703-960f-c1b6d79692f2.png)

And the cameras folder (which has a bunch of folders under it for organization and space):
![Snag_156f26f0](https://user-images.githubusercontent.com/2886615/150619612-41e243d3-d4a4-4c40-b9af-e6305072c4c0.png)

I hope to improve on this over time, and then add some more specific aircraft profiles as I go along. If you have any suggestions or criticisms, I'd appreciate hearing what you have to say.
[Generic Aircraft.zip](https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/files/7917385/Generic.Aircraft.zip)

**Reply by robanderson6287** [reply]
How are you setting up the datarefs to display the value?  For example, I see that you have your flaps page and on the "Flaps Up" it shows a "0" under it.  Same for the fuel, it shows quantity on your screenshot, I haven't found a tutorial on using the plugin for xplane on stream deck.  I downloaded the FlightDeck for X-Plane, it allows me to drag a dataref tile onto the stream deck.  From there I can put a dataref in, it does ask for a value, but it doesn't display anything other than the image for that tile.

**Reply by nguyenquyhy** [reply]
> How are you setting up the datarefs to display the value? For example, I see that you have your flaps page and on the "Flaps Up" it shows a "0" under it. Same for the fuel, it shows quantity on your screenshot, I haven't found a tutorial on using the plugin for xplane on stream deck. I downloaded the FlightDeck for X-Plane, it allows me to drag a dataref tile onto the stream deck. From there I can put a dataref in, it does ask for a value, but it doesn't display anything other than the image for that tile.

FlightDeck author stole our code for commercial purposes without disclosing their source code (as required by GPL license) nor stating the origin of the project. Please take the question to them as we do not support this kind of behavior to open source community.

**Reply by blackcat1101** [reply]
hey dude, would this profile work for the 6 button stream deck?

**Reply by rmroc451** [reply]
> hey dude, would this profile work for the 6 button stream deck?

Individual buttons themselves would be OK. The only thing you can't really use on the 6 button deck is the com/nav/xpdr button, because the additional profile packaged with the plugin doesn't have a 6 button variant.

**Reply by JohnGlossop** [reply]
Newb here. A couple of questions.
1. Will this need mods to work on an XL?
2. Can I use this as the basis for further assignments, or are these settings "locked in". I want to learn how to program the beast but I really don't want it to look like the cockpit for a particular plane, but use it for any plane I fly. I envisage a "front page" allowing me to select a generic type of plane, then build the navigation to suit that type of plane - e.g., single engine simple, single engine complex, ditto twin engine, ditto jet engines, and possibly further sets for 4 engine planes, prop & jet. Oh, and helicopters, as well. Is this doable?

I'd like this as a single "pack" if that's possible, so that I don't have to switch between profiles when I change planes.
I know that looks ambitious, but is it actually feasible with the XL Stream Deck?

**Comment by scuba1976** [comment]
Hi,
I think you can use it, or, at least, use single button as you prefer.
--
Inviato da Libero Mail per Android mercoledì, 20 aprile 2022, 07:48AM +02:00 da blackcat1101  ***@***.*** :

**Comment by grimsan55** [comment]
Any profiles made for the PMDG 737 ?

**Reply by clecomte01** [reply]
look here for msfs streamdeck profiles 

https://flightsim.to/c/third-party/stream-deck/

**Reply by grimsan55** [reply]
@clecomte01 cant find any for pmdg that work with this plugin

**Comment by robanderson6287** [comment]
If you have Spad.Next, the Stream Decks are recognized and you can assign all buttons to whatever you want.  I also use X-Plane and am able to display dataref information on the buttons.  I can have the buttons change color or change image depending on the dataref values.  I haven't really used Microsoft Flight Simulator (I have it and have played around with it) but I do know you can assign buttons to change the screens on the Stream Deck so you can basically have a screen for lighting, autopilot or whatever you want.  You just need to know how to use Spad.Next to take full advantage of what it can do.  Don't use the Stream Deck app, it won't work properly with the sim.
